game-core/src/main/java/games/strategy/triplea/ai/AbstractAi.java

```diff
@@ -477,15 +474,15 @@ public final void start(final String name) {
       politicalActions();
     } else if (name.endsWith("Place")) {
       final IAbstractPlaceDelegate placeDel = (IAbstractPlaceDelegate) getPlayerBridge().getRemoteDelegate();
-      place(name.indexOf("Bid") != -1, placeDel, getGameData(), id);
+      place(name.contains("Bid"), placeDel, getGameData(), id);
     } else if (name.endsWith("EndTurn")) {
       endTurn((IAbstractForumPosterDelegate) getPlayerBridge().getRemoteDelegate(), getGameData(), id);
     }
   }
 
-  /************************
-   * The following methods are called when the AI starts a phase.
-   *************************/
+  /***********************
+   The following methods are called when the AI starts a phase.
```

Comment:
this is not well formatted anymore.

'block', or class structural commentary like this is often a smell, I suggest looking into removing it completely and seeing if we can make the method grouping more obvious.